WcaOnRails/app/models/team.rb

```diff
@@ -59,6 +59,10 @@ def self.wst
     Team.find_by_friendly_id!('wst')
   end
 
+  def self.banned
+    Team.find_by_friendly_id!('banned')
```

Comment:
Does this need to be updated in light of https://github.com/thewca/worldcubeassociation.org/pull/3132?

Reply:
Yes, certainly! Let me know when I should update this PR after #3132 is done. Currently I am OK with waiting to merge this PR, as we only need it after I started to create the actual WDC page.